### ISSUE TITLE
make the gui interface far more accessible to screen readers

### ISSUE THREAD

**Issue by mhussaincov** [himselfv/cbsenum#7]
hi,
would it be possible to make the interface more accessible?
all of the menu options are accessible,
the list of packages are not accessible using nvda the free screen reader or windows narrator.
the search box is accessible but does not have a lable on it spesifieing what it is,
would also be greate if offline support could somehow be added like how install_wim_tweak works?
link to install_wim_tweak:
https://github.com/shiitake/win6x_registry_tweak
link to nvda the screen reader
https://www.nvaccess.org/

Majid
